### ISSUE TITLE
Loader work process failed: scroll the line for preview file (Percent: 0)

### ISSUE THREAD

**Issue by issadarkthing** [kevinhwang91/rnvimr#26]
I have installed all dependencies except ueberzeg but it displays this error whenever i open ranger
![image](https://user-images.githubusercontent.com/50593529/82831364-d3baec00-9eea-11ea-9980-c496096d36e3.png)

**Comment by kevinhwang91** [comment]
Please post your msg by running `nvim +'checkhealth rnvimr'` in shell.

Does this error reproduce by running ranger outside neovim?

**Comment by kevinhwang91** [comment]
Please post your msg by running `nvim +'checkhealth rnvimr'` in shell.

**Comment by issadarkthing** [comment]
`nvim + checkhealth rnvimr`

```health#rnvimr#check
========================================================================
## OS
  - OK: Name: Linux

## Ranger
  - OK: Version: ranger 1.9.2

## Python
  - OK: Version: 3.7.5 (default, Apr 19 2020, 20:18:17) [GCC 9.2.1 20191008]

## Pynvim
  - OK: Version: 0.4.1

## Ueberzug (optional)
  - WARNING: Ueberzug can not calibrate properly outside Tmux.
    - ADVICE:
      - If use Rnvimr without Tmux, please upgrade the latest version of Ueberzug

## RPC
  - OK: Install lib for checkhealth successfully
  - OK: RPC echo: Neovim send "Give me five!" and receive "Give me five!"
  - OK: Clean lib for checkhealth successfully
```

`rc.conf`

```set draw_borders true
set preview_script ~/.config/ranger/scope.sh
map yp shell -d echo -n %d/%f | xsel -i -p -b
map yn shell -d echo -n %f | xsel -i -p -b
default_linemode devicons
set mouse_enabled true
set display_size_in_status_bar true
set colorscheme solarized
map / fzf_select
```

Message log `W`

![image](https://user-images.githubusercontent.com/50593529/82863827-afe0c000-9f55-11ea-9430-4bf06e9579ce.png)

**Comment by kevinhwang91** [comment]
TLDR: update the latest version (1.9.3 for now) of the ranger.

I can reproduce this error in ranger 1.9.2, please update the version of the ranger. I will update the README for at least a version of the ranger. 

This plugin is written since 2020, I don't think to make rnvimr be compatible with the older version of the ranger is a good idea. (1.9.2 released at 2018-09-09)
